books/RayTracingInOneWeekend.html

```diff
@@ -1907,20 +1928,82 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [vec-rand-util]: <kbd>[vec3.h]</kbd> vec3 random utility functions]
 
+</div>
+
+<div class='together'>
+Then we need to figure out how to manipulate a random vector so that we only get results that are on
+the surface of a hemisphere. There are analytical methods of doing this, but they are actually
+surprisingly complicated to understand, and quite a bit complicated to implement. Instead, we'll use
+what is typically the easiest algorithm: A rejection method. A rejection method works by generating
+random samples and seeing if they fall within a bounds of our choosing. If they do, then we
+end the function and return our result. If they fall outside the bounds then we just generate
```

Comment:
~If they fall outside the bounds then we just generate another random sample and see if this one falls into the bounds.~ In other words, keep repeating until you find the solution you want.